sdks/builds/wasm.mk

```diff
@@ -0,0 +1,53 @@
+
+WASM_INTERP_CONFIGURE_FLAGS = \
+	--cache-file=$(TOP)/sdks/builds/wasm-interp.config.cache \
+	--prefix=$(TOP)/sdks/out/wasm-interp \
+	--enable-wasm \
+	--enable-interpreter \
+	--disable-mcs-build \
+	--disable-nls \
+	--disable-boehm \
+	--disable-btls \
+	--with-lazy-gc-thread-creation=yes \
+	--with-libgc=none \
+	--disable-executables \
+	--disable-support-build \
+	--disable-visibility-hidden \
+	--enable-minimal=ssa,com,jit,reflection_emit_save,reflection_emit,portability,assembly_remapping,attach,verifier,full_messages,appdomains,security,sgen_remset,sgen_marksweep_par,sgen_marksweep_fixed,sgen_marksweep_fixed_par,sgen_copying,logging,remoting,shared_perfcounters \
+	--host=i386-apple-darwin10
+
+#toolchain code
+.stamp-wasm-toolchain:
```

Comment:
Why do we have a `wasm` while all other targets are `wasm-interp`?

Reply:
cuz the other wasm targets require their own toolchains.